### PR TITLE
add entity

### PR DESCRIPTION
**Pull request by YanEfimov**

### CODE REVIEW

File: alfaCore/src/main/java/by/st/entity/Voice.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import java.util.Set;
+
+@Entity
+@Table(name = "voice")
+@Getter
+@Setter
+public class Voice {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long voice_id;
```
Comment (vladstepway) [inline alfaCore/src/main/java/by/st/entity/Voice.java]: Not corresponds with code conventions. Make it like "camelCase"

File: alfaCore/src/main/java/by/st/entity/User.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import java.util.Set;
+
+@Entity
+@Table(name = "user")
+@Getter
+@Setter
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long user_id;
```
Comment (vladstepway) [inline alfaCore/src/main/java/by/st/entity/User.java]: Not corresponds with code conventions. Make it like "camelCase"

File: alfaCore/src/main/java/by/st/entity/Answer.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name="answer")
+@Getter
+@Setter
+public class Answer {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long ans_id;
```
Comment (vladstepway) [inline alfaCore/src/main/java/by/st/entity/Answer.java]: Not corresponds with code conventions. Make it like "camelCase"

File: alfaCore/pom.xml [hunk 4]
```diff
@@ -26,10 +26,21 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.16.16</version>
```
Comment (vladstepway) [inline alfaCore/pom.xml:32]: По какому принципу выбиралась версия ?

Comment (YanEfimov) [reply alfaCore/pom.xml:32]: по актуальности

**Pull request merged**